<launch>

  <!--	Arguments	 -->

  <arg name="use_ur_real_robot" default="false"/>
  <arg name="topic_names_file"  default="$(find trajectory_recorder)/config/ur10e_topic_names.yaml"/>


  <!--  Admittance Controller Node  -->
  
  <node pkg="trajectory_recorder" name="trajectory_recorder_Node" type="trajectory_recorder_Node" output="screen">
    <param name="use_ur_real_robot" value="$(arg use_ur_real_robot)"/>
    <rosparam file="$(arg topic_names_file)" command="load"/>
	</node>

</launch>
